plugins/main/public/components/overview/vulnerabilities/dashboards/inventory/inventory.tsx

```diff
@@ -203,18 +200,18 @@ const InventoryVulsComponent = () => {
                       <HitsCounter
                         hits={results?.hits?.total}
                         showResetButton={false}
-                        onResetQuery={() => {}}
+                        onResetQuery={() => { }}
```

Comment:
Should nothing be done in onResetQuery?

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6534/commits/6198d115e0c926270c73f4b841cb937ea14d6153